scrapers/ProducersFun.yml

```
@@ -8,7 +8,7 @@ xPathScrapers:
   sceneScraper:
     scene:
       Title: //div[@class="shadow video-details"]/h1/text()
-      Details: (//div[@class="shadow video-details"]/p[not(@class)])[1]/text()
+      Details: (//div[@class="shadow video-details"]/p[not(@class)])[1]/text()[3]
```

Does not work with `https://producersfun.com/video/chanel-grey-a-fucking-conversation`
If there was an issue with the existing selector just use the below
```suggestion
      Details: (//div[@class="shadow video-details"]/p[not(@class)])[1]
```
We select the whole `<p>` and let stash get the inner text